.github/workflows/ci-kube-prometheus.yml

```
@@ -0,0 +1,133 @@
+              "kubernetes-system-kube-proxy") echo "kubeProxy" ;;
+              "kubernetes-system-scheduler") echo "kubeSchedulerRecording" ;;
+              "node-exporter") echo "nodeExporterAlerting" ;;
+              "node-exporter.rules") echo "nodeExporterRecording" ;;
+              "node-network") echo "network" ;;
+              *) echo "" ;;
+            esac
+          }
+          
+          mkdir -p "${GITHUB_WORKSPACE}/charts/bitnami/kube-prometheus/templates/prometheus/rules"
+          cd "${GITHUB_WORKSPACE"/charts/bitnami/kube-prometheus/templates/prometheus/rules"
+          helm repo add prometheus-community https://prometheus-community.github.io/helm-charts 2>&1 >/dev/null 2>&1
+          helm repo update >/dev/null 2>&1
+          helm template prometheus-community/kube-prometheus-stack \
+            --set fullnameOverride=foo \
+            --set defaultRules.create=true \
+            --show-only templates/prometheus/rules-1.14/* \
+            | awk 'NF' | yq --no-doc -s '.metadata.name | sub("^foo-"; "") + ".yaml"'
+          for m in *.yaml; do
+            rule_name="${m%.yaml}"
+            rule_parameter="$(rule_parameter_from_rule_name "$rule_name")"
+            # We're just interested in the .spec given we build apiVersion, kind and metadata
+            # based in Bitnami standards
+            spec="$(yq '{"spec": .spec}' "$m")"
+            cat > "$m" << EOF
+          {{- /*
+          Copyright Broadcom, Inc. All Rights Reserved.
+          SPDX-License-Identifier: APACHE-2.0
+          */}}
+
+          {{- if and .Values.prometheus.enabled .Values.prometheus.defaultRules.create .Values.prometheus.defaultRules.rules.$rule_parameter }}
+          apiVersion: monitoring.coreos.com/v1
+          kind: PrometheusRule
+          metadata:
+            name: {{ printf "%s-$rule_name" (include "kube-prometheus.prometheus.fullname" .) }}
+            namespace: {{ include "common.names.namespace" . | quote }}
+            labels: {{ include "kube-prometheus.prometheus.labels" . | nindent 4 }}
+            {{- if .Values.commonAnnotations }}
+            annotations: {{- include "common.tplvalues.render" ( dict "value" ..Values.commonAnnotations "context" . ) | nindent 4 }}
+            {{- end }}
+          $spec
+          {{- end }}
+          EOF
+          done
+
+          cd "${GITHUB_WORKSPACE}/charts"
+          if git status -s | grep "bitnami/kube-prometheus/templates/prometheus/rules"; then
+            git add "bitnami/kube-prometheus/templates/prometheus/rules" && git commit -m "Update Prometheus rules" --signoff
+          fi
+      - id: update-pr
+        name: Push changes
+        run: |
+          cd $GITHUB_WORKSPACE/charts
+          # Push all the new commits, if any
+          if [[ $(git cherry -v) ]]; then
+            git push
+            echo "result=ok" >> $GITHUB_OUTPUT
+          else
+            echo "result=skip" >> $GITHUB_OUTPUT
+          fi
```

You don't need to write these to the output of the job, there is no other job which needs this one. Did you mean something like this to add a notice?

```suggestion
          if [[ $(git cherry -v) ]]; then
            git push
            echo "::notice::Default rules have been updated"
          else
            echo "::notice::No changes made"
          fi
```

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-notice-message

---

We can even removed it, I just copy-pasted from the CI workflow